docs/tutorials/standalone/Bucket/bucket_tutorial.jl

```diff
@@ -168,6 +168,17 @@ import ClimaLand
 include(joinpath(pkgdir(ClimaLand), "parameters", "create_parameters.jl"));
 earth_param_set = create_lsm_parameters(FT);
 
+# Set up the model domain. At every surface coordinate point, we'll solve
+# an ODE for `W` and `Ws`, and for every subsurface point, we solve for `T`.
+# In coupled simulations run at the same
+# resolution as the atmosphere, the bucket horizontal resolution would match the
+# horizontal resolution at the lowest level of the atmosphere model. In general, however, the two
+# resolutions do not need to match. Here we just set up something
+# simple - a Column.
+soil_depth = FT(3.5);
+bucket_domain = Column(; zlim = (-soil_depth, FT(0.0)), nelements = 10);
+surface_space = bucket_domain.space.surface
+
 # Define our `BulkAlbedoFunction` model using a constant surface and snow albedo:
 # The surface albedo is a function of coordinates, which would be
```

Comment:
small change - should these `surface` references be `bareground` now?

Reply:
good point! I'll change it in all the drivers + tests